docs/onduty/onduty.md

```diff
@@ -0,0 +1,54 @@
+# Community On-Duty
+
+Community on-duty is a weekly rotation of Crossplane community members who have
+volunteered to help the Crossplane community. Volunteers are on-duty for a week
+at a time. The goals of the on-duty rotation are to:
+
+1. Ensure the Crossplane community feels heard and supported.
+1. Prevent issues and pull requests (PRs) from falling through the cracks.
+1. Make it easier for the next person on-duty to achieve the above goals.
+
+Volunteers should be able to commit to focus on being on-duty for the majority
+of their typical working day (e.g. ~7 hours a day). On-duty is expected to
+respond to questions, issues and pull requests at their earliest convenience,
+within 24 hours.
+
+> Don't hesitate to reach out to the current person on-duty if you're interested
+> in volunteering but can't commit to so much time - we may be able to work out
+> smaller shifts.
+
+When on-duty, you should:
+
+* Make sure the title of #general in Crossplane Slack indicates that you are the
+  person folks should ping with questions.
+* Monitor Crossplane Slack - particularly #general and #dev - for questions from
+  the community. Encourage folks to raise issues for more involved queries, bug
+  reports, or feature requests.
+* Monitor GitHub for new issues and PRs within the Crossplane org. Perform
+  initial triage - e.g. thank the person for their contribution, update the PR
+  or issue with any appropriate labels. Use your discretion as to whether you
+  should attempt to address the issue or review the PR during your on-duty
+  shift. On-duty is a great time to learn new things and address low hanging
+  fruit, but you're not responsible for resolving everything that comes up
+  during your shift.
+* If there's a release happening during your on-duty week consider volunteering
+  to run (or help with) the release.
+* If there's a release happening the week after your on-duty week please take a
+  pass over any open pull requests and ask the folks working on them whether
+  they're release blockers and whether they expect to have them merged in time
+  for a one week feature freeze before the release.
```

Comment:
Might be worth also mentioning to look at the release project board in case there are work items that do not have a PR open but were added to "Accepted".

Reply:
Somehow that feels like it's juuuust crossing the line from on-duty to release management so I'm on the fence as to whether it's appropriate here. I'm not sure how rational that feeling is though - perhaps it's because we're already expecting on-duty to be aware of open PRs even if they're not doing a release?

If we think we can commit for on-duty to always be the release manager (or at least a shadow) maybe we can just add a little section WRT to releases here and include this?